src/attribute.ml

```diff
@@ -150,11 +149,17 @@ module Context = struct
     | Psig_extension ->
       { x with psig_desc = Psig_extension (get_psig_extension x |> fst, attrs) }
     | Rtag                   ->
-      let (lbl, _, can_be_constant, params_opts) = get_rtag x in
-      Rtag (lbl, attrs, can_be_constant, params_opts)
+      begin match x with
+      | Rtag (lbl, _, can_be_constant, params_opts) ->
+        Rtag (lbl, attrs, can_be_constant, params_opts)
+      | Rinherit _ -> x
```

Comment:
Should we assert that the list of attributes is empty here? Otherwise it is effectively silently dropped